modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc

```diff
@@ -66,6 +66,16 @@ Graviton instance types are only available for new clusters created after 24 Jul
 - x2gd.12xlarge (32 vCPU, 768 GiB)
 - x2gd.16xlarge (64 vCPU, 1024 GiB)
 - x2gd.metal (64 vCPU, 1024 GiB)
+- x8g.xlarge (4 vCPU, 64 GiB)
+- x8g.2xlarge (8 vCPU, 128 GiB)
+- x8g.4xlarge (16 vCPU, 256 GiB)
+- x8g.8xlarge (32 vCPU, 512 GiB)
+- x8g.12xlarge (48 vCPU, 768 GiB)
+- x8g.16xlarge (64 vCPU, 1,024 GiB)
+- x8g.24xlarge (96 vCPU, 1,536 GiB)
+- x8g.48xlarge (192 vCPU, 3,072 GiB)
+- x8g.metal-24xl	 (96 vCPU, 1,536 GiB)
```

Comment:
This renders correctly, but I'm not sure the code is correct with the large space between the instance type and information in parentheses.